### ISSUE TITLE
Running problem

### ISSUE THREAD

**Issue by jingjing-unilu** [PhilippChr/CONVEX#6]
Hi,
I got the problem when running and couldn't get the results.txt file. 
Very confused about this information...Could you provide some hints?
```
Predicate Bitmap in 1 min 59 sec 342 ms 10 us
Count predicates in 7 min 47 sec 408 ms 825 us
Count Objects in 8 min 20 sec 819 ms 140 us Max was: 543914579
Bitmap in 16 sec 259 ms 55 us
Bitmap bits: 7230127313 Ones: 1190479283

```
Best Regards,

**Comment by PhilippChr** [comment]
Hi,
the hdt file (+index) is quite large and the hdt library loads everything into RAM when processing the file. How much RAM does your system have?
You will need at least 100 GB of RAM to run it.
Regards,
Philipp

**Comment by jingjing-unilu** [comment]
> Hi,
> the hdt file (+index) is quite large and the hdt library loads everything into RAM when processing the file. How much RAM does your system have?
> You will need at least 100 GB of RAM to run it.
> Regards,
> Philipp



Hi Philipp,
Many thanks for the reply.^^
I have run it on my 16GB RAM local computer now. XD
There are two methods I'm trying now. 
1)use 1-10% of the hdt file. 
2)use HPC. 

Best regards,

**Comment by jingjing-unilu** [comment]
Hi Philipp,

I have tried a smaller dataset which I get from [http://gaia.infor.uva.es/hdt/](url). 
I use the` wikidata-2014-5-26...` version(**only 2.0 GB**). 
It cost a couple of hours to run it and I got the results finally.^^

Besides, I'm running the code with the **whole dataset** now with 100 GB of RAM setting.

Furthermore, `unzip` doesn't work for me to decompressing the `.gz` file. Thus, I use `gunzip`.

Many thanks for your great help.^^

Best regards,
Jingjing
